tests/test_fieldtypes.py

```diff
@@ -998,5 +999,59 @@ def test_datetime_comparisions():
     assert dt("2023-01-02") != datetime(2023, 3, 4, tzinfo=UTC)
 
 
+def test_command_record():
+    TestRecord = RecordDescriptor(
+        "test/command",
+        [
+            ("command", "commando"),
+        ],
+    )
+
+    record = TestRecord(commando="help.exe -h")
+    assert record.commando.executable == "help.exe"
+    assert record.commando.args == ["-h"]
+
+    record = TestRecord(commando="something.so -h -q -something")
+    assert record.commando.args == ["-h", "-q", "-something"]
+
+
+def test_command_integration(tmp_path):
+    TestRecord = RecordDescriptor(
+        "test/command",
+        [
+            ("command", "commando"),
+        ],
+    )
+
+    with RecordWriter(tmp_path / "command_record") as writer:
+        record = TestRecord(commando=r"\.\?\some_command.exe -h,help \d quiet")
+        writer.write(record)
+        assert record.commando.executable == r"\.\?\some_command.exe"
+        assert record.commando.args == [r"-h,help \d quiet"]
+
+    with RecordReader(tmp_path / "command_record") as reader:
+        for record in reader:
+            assert record.commando.executable == r"\.\?\some_command.exe"
+            assert record.commando.args == [r"-h,help \d quiet"]
+
+
+@pytest.mark.parametrize(
+    "command_string, expected_executable, expected_argument",
+    [
+        ("windows.exe something,or,somethingelse", "windows.exe", ["something,or,somethingelse"]),
```

Comment:
For each line, please state what you test here, i.e. "Can we parse a regular windows command with arguments?".
I also miss a a test for a command without arguments and a test for an illegal value (not the happy-flow).